### PR TITLE
fixed package-info-cache-test to use ${path.sep} instead of / to work on Windows

### PR DESCRIPTION
**Pull request by dcombslinkedin**

### CODE REVIEW

File: tests/unit/models/package-info-cache-test.js [hunk 1]
```diff
@@ -207,7 +208,7 @@ describe('models/package-info-cache.js', function() {
         project = fixturifyProject.buildProjectModel(Project);
         project.discoverAddons();
         pic = project.packageInfoCache;
-        projectPackageInfo = pic.getEntry(`${fixturifyProject.root}/simple-ember-app`);
+        projectPackageInfo = pic.getEntry(`${fixturifyProject.root}${path.sep}simple-ember-app`);
```
Comment (Turbo87) [inline tests/unit/models/package-info-cache-test.js]: we should probably use `path.join(fixturifyProject.root, 'simple-ember-app')` instead

**Comment by stefanpenner** [comment]
getEntry should normalize it’s input, we should not leak platform specific specificier details to the caller.

Instead we should mimic what all fs functions do, and normalize

**Comment by dcombslinkedin** [comment]
@stefanpenner , you're probably right about getEntry, but that also means that the cache keys (paths) need to be normalized on the way in, in several places. I can do it, but it's a bit of work. I'll file a ticket on myself to support that.

**Comment by scalvert** [comment]
This is blocking other PRs. Do you have an ETA for this fix? Asking for a friend... :)

**Comment by twokul** [comment]
@scalvert https://github.com/ember-cli/ember-cli/issues/7998#issuecomment-417422371 😝

**Pull request merged**